src/backend/distributed/utils/citus_nodefuncs.c

```diff
@@ -307,3 +329,57 @@ citus_extradata_container(PG_FUNCTION_ARGS)
 
 	PG_RETURN_NULL();
 }
+
+
+#if (PG_VERSION_NUM >= 90600)
+
+/* *INDENT-OFF* */
+#define DEFINE_NODE_METHODS(type) \
+	{ \
+		#type, \
+		sizeof(type), \
+		NULL, \
+		NULL, \
```

Comment:
Hm, these are actually mandatory, so until we need them, we should probably use the same function in all of them that just errors out.